ui/desktop/electron-app/src/helpers/spawn-promise.js

```diff
@@ -25,9 +25,13 @@ module.exports = {
    * @param {string} command
    * @return {Promise}
    */
-  spawnAsyncJSONPromise(command) {
+  spawnAsyncJSONPromise(command, token) {
     return new Promise((resolve, reject) => {
-      const childProcess = spawn(path(), command);
+      const childProcess = spawn(path(), command, {
+        env: {
+          BOUNDARY_TOKEN: token
```

Comment:
We should probably spread the original `...process.env` so the child processes have the same environment variables

Reply:
Good catch!